conans/client/file_copier.py

```diff
@@ -6,18 +6,23 @@
 from conans import tools
 
 
-def report_copied_files(copied, output, warn=False):
+def report_copied_files(copied, output, warn=None):
```

Comment:
Maybe it is cleaner to return None or False when no files have been copied and print outside the function (removing the warn parameter), because if the message comes from outside, and the need of print or not comes from outside... probably better to print outside.